scriptworker/constants.py

```diff
@@ -205,9 +207,11 @@
                 "schemes": ("https", "ssh", ),
                 "netlocs": ("github.com", ),
                 "path_regexes": (
-                    r"^(?P<path>/mozilla-mobile/focus-android)(/|.git|$)",
-                    r"^(?P<path>/mozilla-mobile/android-components)(/|.git|$)",
-                    r"^(?P<path>/mozilla-mobile/reference-browser)(/|.git|$)",
+                    # XXX We allow Github forks to run staging releases. Please make sure to
+                    # restrict production scopes to the origin repo (with cot_restricted_scopes)!
+                    r"^(?P<path>/[A-Za-z0-9-]+/android-components)(/|.git|$)",
```

Comment:
I checked: github "Username may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen". This regex is a little bit more permissive.

Reply:
Does it make sense to allow any github repository? Forks don't necessarily need to use the same name as the base repository.

Reply:
That's a great point. Allowing all github reposity sounds risky for now. @escapewindow suggested to limit the list down to known users (who are willing to make staging releases). Let's see how this goes.